### DISCUSSION TITLE
Social Image for a repo is cropped in explore lists

### DISCUSSION THREAD

**Discussion by sketchbuch** [community/community#87196] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

If you create a social image for a repo following the instructions here:

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview

Images look fine on sites liked linkedin etc., but here on github the image is cropped in explore lists:

![image](https://github.com/community/community/assets/16445037/c41fff0f-40d1-4806-aa73-35a729ea9792)

**Comment by ghost** [comment]
hi @sketchbuch i too followed the resolution suggested by github for best display...But mine fits fine...Maybe the issue related to browser or with the image...

![Screenshot 2024-01-14 151326](https://github.com/community/community/assets/121353576/d017b484-1726-4c37-a902-85c98d0c77ad)

**Comment by sketchbuch** [comment]
How far from the edges is your text? No mention of this was in the help docs. Maybe mine is too close.

**Reply by ghost** [reply]
wait i will send you both close screenshots

**Comment by ghost** [comment]
Hi @sketchbuch Now I realised actually the preview image is cropped a bit vertically...So you were right from beginning maybe Github need to look onto it...


![Ideas For The Weekend](https://github.com/community/community/assets/121353576/7aa71122-58cb-41da-a1cd-78c87a04abbe)

![Screenshot 2024-01-14 151326](https://github.com/community/community/assets/121353576/81bde9d1-acf6-47c0-8c87-297e7efec544)


So try giving some padding around the Preview image to fit the best...Sorry for the misleading comment i had given..

Thank You
